release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -162,6 +162,12 @@ You are no longer required to configure AWS VPC endpoints when installing a rest
 
 For more information, see xref:../installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc#installation-custom-aws-vpc-requirements_installing-restricted-networks-aws-installer-provisioned[Requirements for using your VPC].
 
+[id="ocp-4-11-capabilities"]
+==== Additional customization when installing {product-title}
+{product-title} 4.11 allows you to disable the installation of the `baremetal` and `marketplace` Operators, and the `openshift-samples` content that is stored in the `openshift` namespace. You can disable these features by adding the `baselineCapabilitySet` and `additionalEnabledCapabilities` parameters to the `install-config.yaml` configuration file prior to installation. If you disable any of these capabilities during the installation, you can enable them after the cluster is installed. Once a capability has been enabled, it cannot be disabled again.
```

Comment:
Wouldn't these be Bare Metal and Market Place Operators, since it's referring to the actual Operator? 

Only use "once" when referred to time: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#once 

Probably swap for "When"

Reply:
> Wouldn't these be Bare Metal and Market Place Operators, since it's referring to the actual Operator?
> 
> Only use "once" when referred to time: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#once
> 
> Probably swap for "When"

I have seen the baremetal and marketplace Operators referred to in a few different ways in various places in the doc - (metal3, baremetal, Bare Metal, Baremetal, marketplace, redhat-marketplace, Red Hat Marketplace, etc). So I decided to refer to them by the same names that will be used in the actual parameters in the `install-config.yaml` file (which I got from the dev doc [here ](https://pkg.go.dev/github.com/openshift/api/config/v1#ClusterVersionCapability)).

Is it okay to say "After a capability is enabled..." to indicate that you can't disable the operator after it's been installed?

Reply:
Regarding the first: Sure. The inconsistency drives me insane (not here, just in general in the docs), but I can see why you chose to put those in backticks since they're part of the config YAML. That sounds good to me.

Regarding the second: Yes that sounds good to me!